cogs/source/source.py

```diff
@@ -23,35 +25,46 @@ def messageBuilder(titleEnglish: str, anilistID: str, episode: str, similarity:
     return message
 
 
-async def postSourceFunction(ctx, imageURL):
```

Comment:
Note to self: we're moving it within the class since we declared single `tracemoepy.AsyncTrace` instance and are reusing it.